frontend/packages/dev-console/src/components/topology/TopologySideBar.tsx

```diff
@@ -39,6 +40,7 @@ const TopologySideBar: React.FC<TopologySideBarProps> = ({ item, show, onClose }
       routes,
       services,
       buildConfigs,
+      pods: _.get(item, 'pods'),
```

Comment:
Shouldn't need `_.get` here. We check if `item` is defined above. We get better type checking with `item.pods`

```suggestion
      pods: item.pods,
```